src/plugin/fdpp/boot.c

```diff
@@ -179,7 +179,7 @@ int fdpp_boot(far_t *plt, int plt_len, const void *krnl, int len,
     env[env_len++] = '\0'; // third terminator is a word, not byte
 
     memcpy(boot_sec + 0x03, "FDPP 1.6", 8);
-    memcpy(boot_sec + FDPP_PLT_OFFSET, plt, sizeof(plt) * plt_len);
+    memcpy(boot_sec + FDPP_PLT_OFFSET, plt, sizeof(*plt) * plt_len);
```

Comment:
Wow!

Reply:
Yes. BTW can you look at CID 425195, it looks like it might be important but expect you'll understand what's going on?

Reply:
I can't find 425195 in the (only 1)
coverity e-mail I got.

Reply:
Oh, if you go to the site https://scan.coverity.com/projects/andrewbird-dosemu2 login (you can use your github id) then press 'view defects' you get far better tool than just the email.

Reply:
I am getting some " Error 1016" by that URL.
Many security-related sites are blocking
IPs from RU.

Reply:
okay, I'll see if there's any way I can export that issue.

Reply:
![Screenshot_2024-04-16_11-31-23](https://github.com/dosemu2/dosemu2/assets/220835/3cc8c757-5569-491c-a7b2-a5c57e9fc571)

Reply:
Should now be fixed, thanks!